### PR TITLE
Add Authentication

### PR DESCRIPTION
**Pull request by Megha-Pathak**
Preview - https://coral-store-git-add-auth-megha-pathak.vercel.app/pages/login.html

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

File: css/form.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.form-page-container {
+	height: fit-content;
+}
+
+.form-page-container .form-container {
+	min-width: 50vmin;
+	margin: 2rem;
+	flex: 1 0 auto;
+    background-color: #fff;
```
Comment (kotalwar-krishna) [inline css/form.css]: if you are using same background color for diffrent pages its better to use var

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge

File: css/form.css (hunk 1 above)
Comment (Megha-Pathak) [reply css/form.css]: Corrected

**Pull request merged**